ggml.c

```diff
@@ -20331,11 +20331,11 @@ size_t ggml_quantize_chunk(
         enum ggml_type   type,
            const float * src,
                   void * dst,
-                   int   start,
-                   int   nrows,
-                   int   n_per_row,
+               int64_t   start,
+               int64_t   nrows,
+               int64_t   n_per_row,
            const float * imatrix) {
-    const int n = nrows * n_per_row;
+    const size_t n = (size_t) nrows * n_per_row;
```

Comment:
The convention is to use `int64_t` to count elements and `size_t` for size in bytes

```suggestion
    const int64_t n = (int64_t) nrows * n_per_row;
```

Reply:
Yep, this was an oversight.

Unless there is measurable performance different, I think it is better to keep everything as `int64`. The reason is that `nrow` and `n_per_row` already come from 64 bit integers in `ggml_tensor::ne`, so passing them as int is downcasting and eventually will lead to more overflow issues.

Reply:
Yes, on second thought I agree better to keep `int64_t`. For performance considerations, probably we have to be more careful in the GPU kernels. On the CPU I don't expect there would be much of a difference between 32-bit and 64-bit ints